Ast/src/Parser.cpp

```diff
@@ -2706,14 +2728,18 @@ AstExpr* Parser::parseInterpString()
             break;
         case Lexeme::BrokenInterpDoubleBrace:
             nextLexeme();
-            return reportExprError(location, {}, ERROR_INVALID_INTERP_DOUBLE_BRACE);
+            return reportExprError(endLocation, {}, ERROR_INVALID_INTERP_DOUBLE_BRACE);
```

Comment:
nit: note for some future clean-up PR, constant should have been a `static const char* kErrorInvalidInterpDoubleBrace = ;` instead of a `define`.

Reply:
For whatever reason, that didn't work when I tried it.

https://github.com/Roblox/luau/pull/614/files#r931152226

Reply:
Another work-around would be `reportExprError(endLocation, {}, "%s", kErrorInvalidInterpDoubleBrace);` but doesn't matter any more.